code/obj/item/clothing/helmets.dm

```diff
@@ -116,30 +116,81 @@
 	desc = "Helps protect against vacuum. Comes in a unique, flashy style."
 
 /obj/item/clothing/head/helmet/space/custom
+
 	name = "bespoke space helmet"
-	desc = "Helps protect against vacuum, and is custom-made just for you!"
-	onMaterialChanged()
-		if(src.material)
-			if(material.hasProperty("thermal"))
-				var/prot = round((100 - material.getProperty("thermal")) / 2)
+	desc = "A custom built helmet with a visor that encompasses most of the face."
+	icon_state = "spacemat"
+
+	inhand_image_icon = 'icons/mob/inhand/hand_headgear.dmi' // inhand shit
+	item_state = "s_helmet"
+
+	icon = 'icons/obj/clothing/item_hats.dmi' // prep the world icon_state for building, is made later
+	var/datum/material/fabr_material = null
+	var/datum/material/visr_material = null
+	var/image/fabrImg = null
+	var/image/visrImg = null
+
+	New()
+		..()
+		fabrImg = image(icon, "spacemat")
+		visrImg = image(icon, "spacemat-vis")
+		overlays += fabrImg
+		overlays += visrImg
+
+	proc/setupVisorMat(var/datum/material/V, var/datum/material/C)
```

Comment:
Modify to only take the material which is not `src.material` here. Move the rest to `onMaterialChanged`.